index.html

```diff
@@ -65,6 +63,8 @@ <h2 class="presentation">
 
     <!-- works -->
     <section id="portfolio" class="works">
```

Comment:
Great job @maov19 🤝

- [ ] As I can see from the previous review comment. You need to stick to the design given in the Figma template as much as possible. At the moment your popup doesn't replicate the design given.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/216432555-ad467b4d-bab8-4522-9f16-b54dd4b971bb.PNG)
- Given design
![given-design](https://user-images.githubusercontent.com/63917725/216432616-42857fe7-80ff-499e-bfe8-0696e76ace01.PNG)

- [ ] [Optional] You may also implement the button interactions of `hover` on your popup windows.